base/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
A mention about why views.py isn't _typically_ used in a Wagtail website could be useful.